source/includes/_events.md.erb

```diff
@@ -0,0 +1,165 @@
+# Events
+
+Events are planned social occasions that tickets will be sold for and people will attend.
+
+An Event belongs to an `Account` and most other resources belong to an event.
+
+An event URL is comprised of the combination of the `account_slug` and the `event_slug`.
+
+For example `https://ti.to/ultimateconf/<%= Date.today.year %>` is composed:
+
+* The base `ti.to` URL
+* The Ultimate Conf `Account` slug `ultimateconf`
+* The specific `Event` slug for the <%= Date.today.year %> edition.
+
+The equivalent event can be looked up using a similar URL structure with the API base URL:
+
+`<%= config[:api_endpoint] %>/ultimateconf/<%= Date.today.year %>`
+
+## Attributes
+
+<%= partial "includes/shared/attributes", locals: { attributes: data.events.attributes } %>
+
+## Get all Events
+
+<%= partial "includes/requests/get",
+  locals: {
+    url: "#{config[:api_endpoint]}/:account_slug/events"
+  }
+%>
+<%= partial "includes/events/index.json" %>
+
+Retrieves all `Events` for the given `Account`.
+
+`<%= "GET #{config[:api_endpoint]}/:account_slug/events" %>`
+
+* `account_slug` - The `slug` attribute of an `Account`.
+
+
+## Get an Event
+
+<%= partial "includes/requests/get",
+  locals: {
+    url: "#{config[:api_endpoint]}/:account_slug/:event_slug"
+  }
+%>
+<%= partial "includes/events/show.json" %>
+
+Retrieve an individual `Event` for the given `Account`.
+
+`<%= "GET #{config[:api_endpoint]}/:account_slug/:event_slug" %>`
+
+* `account_slug` - The `slug` attribute of an `Account`.
+* `event_slug` - The `slug` attribute of an `Event`.
+
+## Create an Event
+
+<%= partial "includes/requests/post",
+  locals: {
+    url: "#{config[:api_endpoint]}/:account_slug/events",
+    parameters: {
+      event: {
+        title: "Awesome Conf"
+      }
+    }
+  }
+%>
+<%= partial "includes/events/create.json" %>
+
+
+Create a new `Event` belonging to the given `Account`.
+
+`<%= "POST #{config[:api_endpoint]}/:account_slug/events" %>`
+
+* `account_slug` - The `slug` attribute of an `Account`.
+
+### Parameters
+
+<%= partial "includes/shared/parameters", locals: { parameters: data.events.attributes.reject { |name,attribute| attribute["read_only"] } } %>
+
+## Update an Event
+
+<%= partial "includes/requests/patch",
+  locals: {
+    url: "#{config[:api_endpoint]}/:account_slug/:event_slug",
+    parameters: {
+      event: {
+        location: "Dublin, Ireland"
+      }
+    }
+  }
+%>
+
+Update an `Event` belonging to the given `Account`.
+
+`<%= "PATCH #{config[:api_endpoint]}/:account_slug/:event_slug" %>`
+
+* `account_slug` - The `slug` attribute of an `Account`.
+* `event_slug` - The `slug` attribute of an `Event`.
+
+<aside class="warning">
+FIXME: THIS IS MISSING FROM DASHBOARD
+</aside>
```

Comment:
This endpoint doesn't exist for dashboard. The only endpoint I could find that might be the same is `SettingsController#update`. What do we want to do?

Reply:
Adding as a pivotal bug to come back to : https://www.pivotaltracker.com/story/show/161714421